templates/docker-service.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    #Mandatory annotations in order to enable the docker ingress rule (https://github.com/jenkins-x/exposecontroller)
+    fabric8.io/expose: "true"
+    fabric8.io/exposeUrl: https://docker.${NAMESPACE}.dev.nuxeo.com
+    #Optional annotations in order to customize the docker ingress rule
+    fabric8.io/ingress.path: "/(.*)"
+    fabric8.io/ingress.tls-secret-name: "${NAMESPACE}-tls" #Configure the TLS certificate to be used by the docker ingress rule
+    #fabric8.io/ingress.annotations: customize the ingress rule to act as a reverse proxy and enable upload or download of large files
+    #  nginx.ingress.kubernetes.io/proxy-body-size:
+    #  nginx.ingress.kubernetes.io/rewrite-target:
```

Comment:
do we want to keep these commented lines?

Reply:
Yes, we can remove these commented lines.